src/popup/components/Modals/ConfirmRawSign.vue

```diff
@@ -83,11 +83,7 @@ export default defineComponent({
 
     const activeAccount = getLastActiveProtocolAccount(PROTOCOL_AETERNITY);
 
-    const dataAsString = computed(
-      (): string => (typeof popupProps.value?.data === 'string')
-        ? popupProps.value?.data
-        : Buffer.from(popupProps.value?.data as any).toString('hex'),
-    );
+    const dataAsString = computed((): string => popupProps.value?.txBase64?.toString() || '');
```

Comment:
As there is no such a thing as `data` anymore we were able to skip any conditional checks.